VirtoCommerce.Storefront/Domain/Stores/StoreService.cs

```diff
@@ -80,7 +80,7 @@ protected virtual async Task<Store> ConvertStoreAndLoadDependeciesAsync(StoreApi
                     .Select(pm =>
                     {
                         var paymentMethod = pm.ToStorePaymentMethod(currency);
-                        paymentMethod.Name = pm.TypeName;
+                        paymentMethod.Name = pm.Name;
```

Comment:
This is breaking changes for other themes,  I suggest adding a new field `DisplayName`  and left the old `Name` as is.